src/spec_tools/json_schema.cljc

```diff
@@ -21,21 +21,33 @@
 (defn- spec-dispatch [dispatch spec children] dispatch)
 (defmulti accept-spec spec-dispatch :default ::default)
 
+;;
 ;; predicate list taken from https://github.com/clojure/clojure/blob/master/src/clj/clojure/spec/gen.clj
+;;
 
 ; any? (one-of [(return nil) (any-printable)])
+(defmethod accept-spec 'clojure.core/any? [_ _ _] {:type "object"})
+
 ; some? (such-that some? (any-printable))
+(defmethod accept-spec 'clojure.core/some? [_ _ _] {:type "object"})
+
 ; number? (one-of [(large-integer) (double)])
+(defmethod accept-spec 'clojure.core/number? [_ _ _] {:type "double"})
 
 ; integer? (large-integer)
 (defmethod accept-spec 'clojure.core/integer? [_ _ _] {:type "integer"})
 
 ; int? (large-integer)
-(defmethod accept-spec 'clojure.core/int? [_ _ _] {:type "integer"})
+(defmethod accept-spec 'clojure.core/int? [_ _ _] {:type "integer" :format "int64"})
```

Comment:
`:format` is actually Swagger schema stuff, this could go to https://github.com/metosin/spec-swagger/blob/master/src/spec_swagger/schema.cljc. The same applies to all `:format`s below.